src/SymfonyCache/HttpCacheAware.php

```diff
@@ -24,7 +24,7 @@
     private $httpCache;
```

Comment:
then this `@var` should also have `|null`

Reply:
Right, fixed!